bower.json

```diff
@@ -7,6 +7,7 @@
   "dependencies": {
     "chart.js": "https://github.com/nnnick/Chart.js.git#2.0.0-beta2",
     "fontawesome": "4.5.0",
+    "fullcalendar": "2.6.0",
```

Comment:
@WesCoomber when adding ui libraries please add them using bower.
This keeps the libraries and the project source code separate.
And keeps this diffs easy to read. :smile: